bridgetown-core/lib/bridgetown-core/concerns/site/configurable.rb

```diff
@@ -32,81 +37,96 @@ def config=(config)
       @config
     end
 
-    # Returns the FrontmatterDefaults or creates a new FrontmatterDefaults
-    # if it doesn't already exist.
+    # Returns the current instance of {FrontmatterDefaults} or
+    # creates a new instance {FrontmatterDefaults} if it doesn't already exist.
     #
-    # Returns The FrontmatterDefaults
+    # @return [FrontmatterDefaults]
+    #   Returns an instance of {FrontmatterDefaults}
     def frontmatter_defaults
       @frontmatter_defaults ||= FrontmatterDefaults.new(self)
     end
 
-    # Whether to perform a full rebuild without incremental regeneration
+    # Whether to perform a full rebuild without incremental regeneration.
+    # If either +override+["incremental"] or +config+["incremental"] are true,
+    # fully rebuild the site. If not, incrementally build the site.
     #
-    # Returns a Boolean: true for a full rebuild, false for normal build
+    # @param [Hash] override
+    #   An override hash to override the current config value
+    # @option override [Boolean] "incremental" Whether to incrementally build
+    # @return [Boolean] true for full rebuild, false for normal build
     def incremental?(override = {})
```

Comment:
OMG, the Jekyll folks had the documentation for this backwards! We need to fix this…`incremental?` returns true for the "incremental" build, and false for full rebuilds (the default config).

Reply:
Yea I reread this one multiple times, really threw me for a loop.